pkg/apis/lighthouse/v1alpha1/types.go

```diff
@@ -140,6 +140,8 @@ type LighthouseJobSpec struct {
 	// PipelineRunSpec provides the basis for running the test as a Tekton Pipeline
 	// https://github.com/tektoncd/pipeline
 	PipelineRunSpec *tektonv1beta1.PipelineRunSpec `json:"pipeline_run_spec,omitempty"`
+	// PipelineRunParams are the params used by the pipeline run
+	PipelineRunParams []config.PipelineRunParam `json:"pipeline_run_params,omitempty"`
```

Comment:
This just reminds me that I really should make it so that `config` can depend on `v1alpha1` ASAP...

Reply:
Yep, i had to make it the other way around...
In tekton, config is part of the `apis` folder but not tied to a specific version (https://github.com/tektoncd/pipeline/tree/master/pkg/apis).

Reply:
Things are uglier here because of inheriting so much from Prow, and then me trying to move more of the "config" into `LighthouseJob` and insisting on having `DeepCopy` =) I'll clean it up soon.

Reply:
No worries, i'll help if i have time ;)